amundsen_application/static/js/components/common/Alert/AlertIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+const STROKE_COLOR = '#b8072c'; // $red70
+const SIZE = 16;
+
+const AlertIcon: React.FC = () => {
+  return (
+    <svg
```

Comment:
Would it make sense to have a `common/icons` where we start putting new amundsen icons that use inline-svgs?

Reply:
Makes sense! I will move it there!